### PR TITLE
Stop incrementing LMP after a certain point ctd

### LINKED ISSUE #1008: Stop incrementing LMP after a certain point
Thanks for creating an issue! Please fill out this form so we can be sure to have all the information we need, and to minimize back and forth.

* **What are we trying to do?**
One of the most useful features in the app is our `LMP counter`, which given a date of initial call and another date will tell you how many weeks since someone's last menstrual period. However, we don't have it stop ever, so as we reconcile we're starting to see some people who are, like, 60 weeks pregnant, which, yikes. 

We should make it so this stops incrementing at either the date of appointment, or at 35 weeks 0 days. 

* **What feature or behavior is this required for?**
Reducing the cringe factor for our data entry team.

* **How could we solve this issue? (Not knowing is okay!)**
The code for this is in `app/helpers/last_menstrual_period_helper.rb`. Probably it involves adding some special case handling to the methods in there. You can see an example of how we're using it in practice at `app/views/patients/_patient_dashboard.html.erb`.

* **Anything else?**
Stems from #864

### PR DESCRIPTION
**Pull request by colinxfleming**
I rule and have completed some work on Case Manager that's ready for review!

Very small syntactic bump to #1031 to get tests to pass.

This pull request makes the following changes:
* remove `patient` from initial call date ref

It relates to the following issue #s: 
* stems from  #1031 
* Fixes #1008

### CODE REVIEW

**Comment by colinxfleming** [comment]
@DarthHater can I get a quick review on this?

**Comment by colinxfleming** [comment]
(or @KatSDC if you're on the grid this weekend)

File: app/models/concerns/last_menstrual_period_measureable.rb [hunk 1]
```diff
@@ -30,7 +30,11 @@ def _last_menstrual_period_on_date(date)
     return nil unless initial_call_date && last_menstrual_period_weeks
     weeks = 7 * (last_menstrual_period_weeks || 0)
     days = (last_menstrual_period_days || 0)
-    (date - initial_call_date) + weeks + days
+    if (date - initial_call_date) + weeks + days > 280
```
Comment (DarthHater) [inline app/models/concerns/last_menstrual_period_measureable.rb]: Might want to just dry this out into another quick helper method since you repeat it.

Comment (colinxfleming) [inline app/models/concerns/last_menstrual_period_measureable.rb]: Good call! (Also good call on the test, I'll whip one up now)

**Pull request merged**